schemas/dns.yml

```diff
@@ -159,6 +179,66 @@
         be the original `question.name` repeated.
       example: www.google.com
 
+    - name: answers.registered_domain
```

Comment:
I'm not sure I would add all of these fields in the DNS answers array of object.

@MikePaquette Do you think we have a need for that?

@leehinman & @andrewkroh What do you think about implementing this on DNS answers? Sounds to me like adding this here may be problematic, considering the different kinds of DNS answers we can get, or even just for performance reasons: a query for a domain name can yield many CNAMEs as an answer, so worst case this could be multiple public suffix list lookups per DNS answer event. WDYT?

Reply:
Looking at a packetbeat dns request there's 1 request for www.microsoft.com, which gets answered with  3 CNAMES and an A record.    (reference below). As the A record is effectively a recursive, automatic request - it seems like the name should be parsed out and treated like a straight up dns.question??

> {
>   "data": "www.microsoft.com-c-3.edgekey.net",
>   "name": "www.microsoft.com",
>   "type": "CNAME",
>   "class": "IN",
>   "ttl": "2726"
> },
> {
>   "data": "www.microsoft.com-c-3.edgekey.net.globalredir.akadns.net",
>   "name": "www.microsoft.com-c-3.edgekey.net",
>   "type": "CNAME",
>   "ttl": "21204",
>   "class": "IN"
> },
> {
>   "data": "e13678.dspb.akamaiedge.net",
>   "name": "www.microsoft.com-c-3.edgekey.net.globalredir.akadns.net",
>   "type": "CNAME",
>   "ttl": "900",
>   "class": "IN"
> },
> {
>   "data": "23.33.58.14",
>   "name": "e13678.dspb.akamaiedge.net",
>   "type": "A",
>   "class": "IN",
>   "ttl": "20"
> }